docs/test-conf.md

```diff
@@ -0,0 +1 @@
+# Test configuration
```

Comment:
Fill in with actual documentation, I think we just include it directly in this PR.